index.html

```diff
@@ -24,15 +26,20 @@
         <img src="./images/darwin.jpg" alt="">
         <span>Ramoun</span>
       </a>
-      <nav>
+      <nav class="main-nav">
         <img src="./images/menu.svg" alt="">
+        <ul class="nav-menu space-between cross-center cap">
+          <li><a href="./">home</a></li>
+          <li><a href="./about.html">about</a></li>
+          <li><a href="./contact.html">contact</a></li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] It would be amazing if you add the transition to the other sections, you can put inside the `href` attribute the next value `#id-section`, for instance:
```
<a href="#id-section></a> // Link to go to the #id-section.

<section id="id-section">
</section>
```

- [ ]  [OPTIONAL] You can try to use the `max-width` CSS attribute to avoid this issue for the content for your links and logo on large screens. The red line is the current width, and the blue line is the width expected.

![image](https://user-images.githubusercontent.com/60164239/130311160-662f878e-09ce-4e18-823f-af29d2409b69.png)